src/components/PageSections/Menu/_FullSiteMenu.scss

```diff
@@ -207,14 +225,10 @@
 }
 
 .coa-MenuInfo__resource {
-  // techdebt: we may be dealing with some weird div sizing here
-  // the links weren't clickable without this fix
   z-index: 4;
```

Comment:
do we still need the z-index for the divs to be clickable? if so, could you leave some note for future devs as to why we need the z-index?

Reply:
I think that note got inadvertently swept up when i was cleaning out some noted out code. Didn't mean to remove it. So, good catch! I'll pop it back in!